src/no/runsafe/UserControl/database/PlayerKickLog.java

```diff
@@ -22,22 +25,39 @@ public ISchemaUpdate getSchemaUpdateQueries()
 		update.addQueries(
 			"CREATE TABLE player_kick_log (" +
 				"`name` varchar(255) NOT NULL," +
-				"`timestamp` datetime NULL," +
+				"`timestamp` datetime NOT NULL," +

```

Comment:
It looks like it was like that since it was created, but when testing that mysql query it doesn't seem to work when the timestamp is set to NULL